source/includes/extracts-4.2-changes.yaml

```diff
@@ -0,0 +1,842 @@
+ref: 4.2-changes-index-names
```

Comment:
Hm - given that the new tools are 4.4 specific, how often are we referencing this content? I don't see a lot of refs linking here.

Reply:
ping on this - what happens if we remove this file?

Reply:
cc @andf-mongodb

Reply:
It's referenced 9 times. 1:1 transfer for now imo.

Reply:
Ah damn. We'll tidy up later then.